x/qgb/orchestrator/relayer.go

```diff
@@ -86,6 +86,7 @@ func (r *relayer) updateValidatorSet(
 		return err
 	}
 
+	// this should probably be valset with nonce -1
```

Comment:
Isn't nonce an unsigned int?
```suggestion
	// TODO should this be valset with nonce -1?
```

Reply:
Add a TODO and the edit pls